From: Colin Ian King <colin.i.king@gmail.com>
Subject: selftest-mm-ksm_functional_tests-extend-test-case-for-ksm-fork-exec-fix
Date: Tue, 2 Apr 2024 09:15:37 +0100

There is a spelling mistake in a ksft_test_result_skip message. Fix it.

Link: https://lkml.kernel.org/r/20240402081537.1365939-1-colin.i.king@gmail.com
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Cc: Jinjiang Tu <tujinjiang@huawei.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/ksm_functional_tests.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/ksm_functional_tests.c~selftest-mm-ksm_functional_tests-extend-test-case-for-ksm-fork-exec-fix
+++ a/tools/testing/selftests/mm/ksm_functional_tests.c
@@ -502,7 +502,7 @@ static void test_child_ksm_err(int statu
 	else if (status == -2)
 		ksft_test_result_fail("Merge in child failed\n");
 	else if (status == -3)
-		ksft_test_result_skip("Merge in child skiped\n");
+		ksft_test_result_skip("Merge in child skipped\n");
 }
 
 /* Verify that prctl ksm flag is inherited. */
_
